images/k8s-cloud-builder/test.yaml

```diff
@@ -41,6 +41,12 @@ commandTests:
   - go
   - gsutil
   - yq
+  # for multiarch support / cross building
+  - gcc
+  - arm-linux-gnueabihf-gcc
+  - aarch64-linux-gnu-gcc
+  - powerpc64le-linux-gnu-gcc
+  - s390x-linux-gnu-gcc
   setup:
   -
```

Comment:
This line seems weird. Does it work as intended?

Reply:
Yep, that's intended: `setup` is supposed to be an array of arrays.

Reply:
![Ancient Aliens 20012020180654](https://user-images.githubusercontent.com/567897/72763021-c9d95300-3baf-11ea-8255-d9d8ad123cdc.jpg)

Reply:
We can also switch to json if you prefer that:

```json
{"schemaVersion":"2.0.0","commandTests":[{"name":"python version is 3","command":"python","args":["--version"],"expectedOutput":["^Python 3\\."]},{"name":"no python2 is installed","command":"bash","args":["-c","python2"],"expectedError":["command not found"],"exitCode":127},{"name":"pip is running for python3","command":"pip","args":["--version"],"expectedOutput":["^pip.*python 3\\."]},{"name":"bash is 4+","command":"bash","args":["--version"],"expectedOutput":["version [45]\\."]},{"name":"google sdk is in the $PATH","command":"bash","args":["-c","echo \"$PATH\" | tr : '\\n' | grep '^/opt/google/google-cloud-sdk/bin$'\n"]},{"name":"crcmod python module is installed","command":"bash","args":["-c","pip list | grep '^crcmod\\b'\n"]},{"name":"commands are available","command":"/test.sh","args":["envsubst","gcloud","go","gsutil","yq","gcc","arm-linux-gnueabihf-gcc","aarch64-linux-gnu-gcc","powerpc64le-linux-gnu-gcc","s390x-linux-gnu-gcc"],"setup":[["bash","-c","cat <<'EOF' >/test.sh\n#!/usr/bin/env bash\nset -e\nset -u\nset -o pipefail\n\nexitCode=0\nfor cmd in \"$@\"\ndo\n  if command -v \"$cmd\" >/dev/null 2>&1\n  then\n    echo -n .\n  else\n    exitCode=1\n    >&2 echo \"${cmd} NOT available.\"\n  fi\ndone\nexit $exitCode\nEOF\n\nchmod +x /test.sh\n"]]}]}
```
😏 😝

Reply:
plz no :sweat_smile: